yt_dlp/extractor/twitch.py

```diff
@@ -392,7 +425,13 @@ def _extract_info(self, info):
             'was_live': True,
         }
 
-    def _extract_moments(self, info, item_id):
+    def _extract_chapters(self, info, item_id):
+        if not info.get('moments'):
+            game = traverse_obj(info, ('game', 'displayName'))
+            if game:
+                yield {'title': game}
+            return
+
         for moment in info.get('moments') or []:
```

Comment:
```suggestion
        for moment in info['moments']:
```
since the other cases are already handled